config/releases.exs

```diff
@@ -31,6 +31,24 @@ defmodule Environment do
       _ -> nil
     end
   end
+
+  def get_local_url(key) do
```

Comment:
I think the function could be a bit smaller and the error message more helpful:

```elixir
url = get(key)

case URI.parse(url).host do
  host when host in ~w(localhost 127.0.0.1) -> url
  host -> raise "Expected host of the DATABASE_URL environment variable to be localhost or 127.0.0.1, got: #{host}"
end
```

Reply:
If we’re going with this, I suggest

```elixir
  host -> raise "Expected host of the #{key} environment variable to be `localhost` or `127.0.0.1`, got: #{host}"
```